cmd/gardener-controller-manager/app/gardener_controller_manager.go

```diff
@@ -228,24 +229,37 @@ func NewGardener(config *componentconfig.ControllerManagerConfiguration) (*Garde
 	// Prepare a Kubernetes client object for the Garden cluster which contains all the Clientsets
 	// that can be used to access the Kubernetes API.
 	var (
-		kubeconfig       = config.ClientConnection.KubeConfigFile
-		runningInCluster = kubeconfig == ""
+		kubeconfig         = config.ClientConnection.KubeConfigFile
+		gardenerKubeConfig = config.GardenerClientConnection.KubeConfigFile
+		runningInCluster   = kubeconfig == ""
 	)
+
 	k8sGardenClient, err := kubernetes.NewClientFromFile(kubeconfig)
 	if err != nil {
 		return nil, err
 	}
+
+	gardenClient := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
```

Comment:
Can we call this `gardenerClient`? Sorry, we just recently renamed everything to `gardener-*`, so `gardener-apiserver` and `gardener-controller-manager`.